tools/build_android/android_build.sh

```diff
@@ -13,14 +13,14 @@ export JAVA_HOME="$(/usr/libexec/java_home -v 1.8)" || export JAVA_HOME="/usr/li
 export JAVA_HOME="$(/usr/libexec/java_home -v 1.8)"
 
 export ARCH=$1
-if [ "${ARCH}" == 'armeabi-v7a' ]; then
-	export TOOLCHAIN_NAME='android-ndk-r16b-api-21-armeabi-v7a-clang-libcxx14'
-elif [ "${ARCH}" == 'arm64-v8a' ]; then
-	export TOOLCHAIN_NAME='android-ndk-r16b-api-24-arm64-v8a-clang-libcxx14'
-else
-	export TOOLCHAIN_NAME='android-ndk-r16b-api-16-x86-clang-libcxx14'
+if [ "$ARCH" == "armeabi-v7a" ]; then
+    export TOOLCHAIN_NAME='android-ndk-r16b-api-21-armeabi-v7a-clang-libcxx14'
```

Comment:
Are those tabs? 😢

Reply:
:trollface: I'll fix it

Reply:
Actually I've just checked it seems to be spaces 🤔